templates/web/pages/dashboard/display_orgs_table.tt.html

```diff
@@ -20,8 +23,11 @@
                 <td>[% org.name %]</td>
                 <td>[% org.creator %]</td>
                 <td>[% org.org_id %]</td>
-                <td>[% IF org.validated == 0 %]false[% ELSE %]true[% END %]</td>
+                <td>[% org.valid_org %]</td>
                 <td>[% display_date_tag(org.created_t) %]</td>
+                <td>[% display_date_tag(org.last_logged_member_t) %]</td>
+                <td>[% org.last_import_type %]</td>
+                <td>[% org.creator %]</td>
```

Comment:
It seems that you are displaying the creator instead of the data source of the last import.